editor/editor_file_dialog.cpp

```diff
@@ -1184,10 +1203,39 @@ void EditorFileDialog::_update_drives(bool p_select) {
 }
 
 void EditorFileDialog::_favorite_selected(int p_idx) {
-	dir_access->change_dir(favorites->get_item_metadata(p_idx));
-	update_dir();
-	invalidate();
-	_push_history();
+	Error change_dir_result = dir_access->change_dir(favorites->get_item_metadata(p_idx));
+	if (change_dir_result != OK) {
+		error_dialog->set_text(TTR("Favorized folder does not exist anymore and will be removed."));
+		error_dialog->popup_centered(Size2(250, 50) * EDSCALE);
+
+		bool res = access == ACCESS_RESOURCES;
```

Comment:
```suggestion
		bool res = (access == ACCESS_RESOURCES);
```
To be consistent with the style change done below.